### PR TITLE
[Keyboard] RS108 Addition

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Add RS108 Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add RS108 Keyboard

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xelus/rs108/chconf.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This file was auto-generated by:
+ *    `qmk chibios-confmigrate -i keyboards/acheron/austin/chconf.h -r platforms/chibios/common/configs/chconf.h`
+ */
+
+#pragma once
```
Comment (drashna) [inline keyboards/xelus/rs108/chconf.h:22]: Is this file actually needed, or just copy and pasted?

**Review by drashna — APPROVED** [review]
Aside from the chconf.h file, LGTM.

File: keyboards/xelus/rs108/chconf.h (hunk 1 above)
Comment (Xelus22) [reply keyboards/xelus/rs108/chconf.h:22]: I can remove this and test

**Review by tzarc — APPROVED** [review]
